server/src/main/java/com/lunchtime/service/impl/RestaurantServiceImpl.java

```diff
@@ -13,10 +14,16 @@
 @Service
 public class RestaurantServiceImpl implements RestaurantService {
     private final RestaurantRepository restaurantRepository;
+    private final PersonRepository personRepository;
     private final PersonService personService;
+    private static final long OWNER_ROLE_ID = 2;
 
-    public RestaurantServiceImpl(RestaurantRepository restaurantRepository, PersonService personService) {
+    public RestaurantServiceImpl(
+        RestaurantRepository restaurantRepository,
+        PersonRepository personRepository,
+        PersonService personService) {
         this.restaurantRepository = restaurantRepository;
+        this.personRepository = personRepository;
         this.personService = personService;
     }
```

Comment:
**[@RequiredArgsConstructor](https://projectlombok.org/features/constructor)** will help us to make code more readable. We need just `@RequiredArgsConstructor` annotation and no all args constructor here, as `@RequiredArgsConstructor` automatically generates constructor for `final` fields.